src/test/java/pl/project13/maven/git/GitCommitIdMojoTest.java

```diff
@@ -27,6 +27,7 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Paths;
 import java.util.Map;
```

Comment:
Unfortunately the java.nio.file got introduced with JDK 7 and causes the travis build to fail for Java 6.

[ERROR] /home/travis/build/ktoso/maven-git-commit-id-plugin/src/test/java/pl/project13/maven/git/GitCommitIdMojoTest.java:[30,20] package java.nio.file does not exist
[ERROR] /home/travis/build/ktoso/maven-git-commit-id-plugin/src/test/java/pl/project13/maven/git/GitCommitIdMojoTest.java:[224,43] cannot find symbol
symbol  : variable Paths
location: class pl.project13.maven.git.GitCommitIdMojoTest
[ERROR] /home/travis/build/ktoso/maven-git-commit-id-plugin/src/test/java/pl/project13/maven/git/GitCommitIdMojoTest.java:[235,43] cannot find symbol
symbol  : variable Paths
location: class pl.project13.maven.git.GitCommitIdMojoTest